setup.cfg

```diff
@@ -954,6 +950,10 @@ disallow_incomplete_defs = False
 disallow_untyped_defs = False
 disallow_incomplete_defs = False
 
+[mypy-tools.test.test_utils_pagerduty_api]
+disallow_untyped_defs = True
+disallow_incomplete_defs = True
+
```

Comment:
why this? shouldn't we aim for everything new being properly typed?

Reply:
You are right! We've a general "type hints, I don't care" setting for [all our tests](https://github.com/app-sre/qontract-reconcile/blob/master/setup.cfg#L569) and this explict enables the type checking for the pagerduty test file. `disallow = True` :)

Reply:
ahhh, ok... understood 😍 

I hate properties that are not affirmative 😬

Reply:
Me too :)